### DISCUSSION TITLE
armbian-ddbr with an option like -v to verifie if backup is good ?

### DISCUSSION THREAD

**Discussion by okpapt** [ophub/amlogic-s9xxx-armbian#2289] [Amlogic]
Hello,
Sorry for my poor english.

I have old sd card and some news from ali or gearb.
I am not sure all sectors are good.
How can I verify if backup with **armbian-ddbr  b**  is exactly eMMC content ?

Have a good day.

**Comment by ophub** [comment]
https://github.com/ophub/amlogic-s9xxx-armbian/blob/73050287d6618d2f2d47111a28770bf8608f3b82/build-armbian/armbian-files/common-files/usr/sbin/armbian-ddbr#L87-L96

**Reply by okpapt** [reply]
When I search if option verifie backup content versus eMMS source exist, before asking about here, I had reading this source link, but nothing inside about specific option to verifie if what is write in BACKUP-arm-64-emmc.img.gz is exactly intentical than eMMC.

**Comment by ophub** [comment]
你可以了解一下dd命令的工作原理

**Comment by okpapt** [comment]
Hi,
If I take time to translate my local language into English to be understood by the majority of people, you could do the same, it's the least polite thing to do.
